.github/workflows/ci.yaml

```diff
@@ -0,0 +1,95 @@
+name: ci
+on: 
+  push:
+    branches:
+      - master
+  pull_request:
+jobs:
+  test-linux-and-mac:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        go: ['1.13', '1.14']
```

Comment:
Include master too.

Reply:
Master is not currently supported. See the last issue in PR description.